source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -195,7 +198,7 @@ def _filter_similar_journeys(journeys_pool, request, similar_journey_generator):
         if to_be_deleted(j1) or to_be_deleted(j2):
             continue
         if compare(j1, j2, similar_journey_generator):
-            #chose the best
+            # choose the best
```

Comment:
`# choose the best` -> `worst = _get_worst_similar` ?!?

Reply:
Not my comment, just fixing a typo. But if needed, we can find a more suitable comment